src/app/static/src/js/components/runWorkflow/runWorkflowReport.vue

```diff
@@ -318,6 +366,25 @@ export default Vue.extend<Data, Methods, Computed, Props>({
                     this.error = error;
                     this.defaultMessage = "An error occurred fetching run report metadata";
                 });
+        },
+        validateWorkflow() {
+            const formData = new FormData()
+            formData.append("file", this.importedFile)
+            formData.append("git_branch", this.workflowMetadata.git_branch)
+            formData.append("git_commit", this.workflowMetadata.git_commit)
```

Comment:
I'm actually changing endpoint slightly to accept commit and branch as query string rather than in the form data (to be consistent with other endpoints), so just a heads-up that a small change will be required here when that gets merged.
https://github.com/vimc/orderly-web/pull/393

Reply:
> This PR contains changes from https://github.com/vimc/orderly-web/pull/390/files doesn't it? Would normally expect that PR to be merged to master to get a clean diff on this one before request review - so you were just looking for ideas on testing issues just now rather than a full review? Looks like those lines have changed now so issues fixed?

Absolutely, I was trying to find ideas. However, I have just realised there might not be any validation for empty formData.